c/include/cuvs/neighbors/cagra.h

```diff
@@ -156,6 +156,48 @@ cuvsError_t cuvsCagraCompressionParamsCreate(cuvsCagraCompressionParams_t* param
  */
 cuvsError_t cuvsCagraCompressionParamsDestroy(cuvsCagraCompressionParams_t params);
 
+/**
+ * @brief Create CAGRA index parameters similar to an HNSW index with hard M constraint
+ *
+ * This factory function creates CAGRA parameters that yield a graph of the same size
```

Comment:
Is it approximately same size in `bytes`, or logically same size (M==graph_degree/2)?